src/main.js

```diff
@@ -16,6 +16,7 @@ export { default as EcAmountInput } from './components/ec-amount-input';
 export { default as EcBtn } from './components/ec-btn';
 export { default as EcBtnDropdown } from './components/ec-btn-dropdown';
 export { default as EcButtonGroup } from './components/ec-button-group';
+export { default as EcDatepicker } from './components/ec-datepicker';
```

Comment:
we should move this line a little further down to respect the alphabetical order